### DISCUSSION TITLE
Menu button not open by default

### DISCUSSION THREAD

**Discussion by catplotlib** [chakra-ui/chakra-ui#7863] [Ideas]
### Description

When working with the Menu component, I'd prefer if the MenuList stayed open by default. It would be really cool if a feature could be added to that!

### Problem Statement/Justification

At present, the MenuList within Chakra UI's Menu component does not stay open by default. While this design choice may often be desirable to keep the UI clean and uncluttered, there are scenarios where having the MenuList stay open could enhance the user experience. For example, in applications where users need to make multiple selections from a menu or need to see their options constantly, an open MenuList can save users from repeated clicks.

The introduction of a feature that allows developers to control the default state of the MenuList would give them greater flexibility in UI design. Developers could then customize the behavior of the MenuList to better align with the needs and expectations of their users, thereby improving the overall user experience.

Incorporating this feature into Chakra UI could also increase its versatility and applicability to a broader range of projects, potentially expanding its user base. By enabling more customization options for the Menu component, Chakra UI could cater to a more diverse array of design requirements and use cases.

### Proposed Solution or API

N/A

**Comment by segunadebayo** [comment]
This is currently possible with the `defaultIsOpen` or `isOpen` prop on the Menu component

```jsx
<Menu defaultIsOpen>
      <MenuButton as={Button} variant="solid" colorScheme="green" size="sm">
        Open menu
      </MenuButton>
      <MenuList>
        <MenuItem icon={<FaSearch />} command="⌥T">
          Search
        </MenuItem>
        <MenuItem icon={<FaUndoAlt />}>Undo</MenuItem>
        <MenuItem icon={<FaTruck />}>Delivery</MenuItem>
        <MenuItem icon={<FaUnlink />}>
          Unlink
        </MenuItem>
      </MenuList>
    </Menu>
```
